build.gradle

```diff
@@ -97,3 +95,8 @@ idea.project.ipr.withXml { xml ->
 jdks {
     daemonTarget = 17
 }
+
+javaVersions {
+    libraryTarget = 21
```

Comment:
And there is eclipse... which I have no idea about. Perhaps now is the moment to remove support - we do not support eclipse at all internally at Palantir any more.

Reply:
I guess we could make it do the same external JDK spinning up code we will end up doing for IntelliJ